src/main/java/frc/robot/RobotContainer.java

```diff
@@ -84,18 +80,46 @@ public RobotContainer() {
    * edu.wpi.first.wpilibj.Joystick} or {@link XboxController}), and then passing it to a {@link
    * edu.wpi.first.wpilibj2.command.button.JoystickButton}.
    */
-  private void configureButtonBindings() {
+  public void configureButtonBindings() {
+    initalizeDrivetrainCommands();
     initializeClimberCommands();
+    initializeShooter2020Commands();
   }
 
-  private void initializeClimberCommands() {
+  private void initalizeDrivetrainCommands() {
+    if (RobotConstants.get().hasDrivetrain()) {
+      drivetrain = new Drivetrain();
+      drivetrain.setDefaultCommand(new ArcadeDriveCMD(drivetrain,
```

Comment:
If you're also creating the subsystem, maybe a better name would be `initializeDrivetrain` instead of `initializeDrivetrainCommands`. Also, I'm lazy and would prefer to type `initDrivetrain`.